src/composite_backing.jl

```diff
@@ -0,0 +1,159 @@
+"""
+    backing(x)
+
+Returns a version of `x` that is of the type the backing of a `Composite` is.
+i.e. as a `NamedTuple` or `Tuple`.
+
+This is kind of the opposite of `construct` for structs.
```

Comment:
Does it then make sense to call it `deconstruct` or `unconstruct`?

Reply:
No because that means to destroy
https://en.wikipedia.org/wiki/Destructor_(computer_programming)

Reply:
destruct and deconstruct are different words...

Reply:
`destruct` like `self-destruct` or `destruction`

`destructure` I can get behind,. but it already has a kinda specific meaning

Reply:
Yes my suggestion is `deconstruct` which is a different word to either "destruct" or "destructure". But also :shrug: